schema/exampleSchema.json

```diff
@@ -50,6 +50,11 @@
           "type": "string",
           "maxLength": 11
         },
+        "vehicleSlotTypeCode": {
+          "description": "A numeric code representing the Vehicle Slot Type",
```

Comment:
Is this just a value defined by TARS? Is there a defined list of possible numbers?

Reply:
There is a page containing all of them - https://wiki.i-env.net/display/MES/Test+Category+Data

Reply:
Ah, not a trivial list I see.

Reply:
Is the idea that the slot type will be used to help determine the length of the slot then?

Reply:
We will be using it to show the correct slot length (standard, double, extende) There are a couple of scenarios which only apply to certain vehicle slot types